src/using-workflow/using-docker-images.md

```diff
@@ -33,13 +33,9 @@ In order to deploy Docker images, they must conform to the following requirement
 Use `deis create` to create an application on the [controller][].
 
     $ mkdir -p /tmp/example-go && cd /tmp/example-go
-    $ deis create
+    $ deis create example-go --no-remote
     Creating application... done, created example-go
 
-!!! note
-    The `deis` client uses the name of the current directory as the
-    default app name.
-
```

Comment:
@rimusz seems like it'd be good to keep this here, so people know that they don't _have_ to specify the app name if they don't want. What do you think of the below?

```
!!! note
    The `deis` client uses the name of the current directory as the default app name if you don't specify it, as above
```